website/content/en/docs/setup/going-to-prod/architecting.md

```diff
@@ -0,0 +1,281 @@
+
+##### **When Vector Should Replace Agents**
+
+Vector should replace agents performing generic data forwarding functions, such as:
+
+- Tailing and forwarding log files
+- Collecting and forwarding service metrics without enrichment
+- Collecting and forwarding service logs without enrichment
+- Collecting and forwarding service traces without enrichment
+
+Notice that these functions collect and forward existing data, unchanged. There is nothing unique about these functions. Vector should replace these agents since Vector offers better performance and reliability.
+
+##### **When Vector Should Not Replace Agents**
+
+Vector should not replace agents that produce vendor-specific data that Vector cannot replicate.
+
+For example, the Datadog Agent powers [Network Performance Monitoring](https://www.datadoghq.com/product/network-monitoring/network-performance-monitoring/). In this case, the Datadog Agent integrates with vendor-specific systems and produces vendor-specific data. Vector should not be involved with this process. Instead, the Datadog Agent should collect the data and send it directly to Datadog since the data is not one of [Vector’s supported data types](docs/about/under-the-hood/architecture/data-model/).
+
+As another example, the Datadog Agent collects service metrics and enriches them with vendor-specific Datadog tags. In this case, the Datadog Agent should send the metrics directly to Datadog or route them through Vector. Vector should not replace the Datadog Agent because the data being produced is enriched in a vendor-specific way.
+
+More examples are below.
+
+| Agent | Action | Rationale |
+| --- | --- | --- |
+| Datadog Agent | Integrate | The Datadog Agent integrates with more systems, produces better data, and seamlessly integrates with the Datadog platform. |
+| File Beat | Replace | Vector can tail files with better performance, reliability, and data quality. |
+| Fluentbit | Replace | Vector can perform the same functions as Fluentbit with better performance and reliability. |
+| FluentD | Replace | Vector can perform the same functions as Fluentd with significantly better performance and reliability. |
+| Metric Beat | Integrate | Metric Beat integrates with more systems, produces better data, and seamlessly integrates with the Elastic platform. |
+| New Relic Agent | Integrate | The New Relic Agent integrates with more systems, produces better data, and seamlessly integrates with the New Relic platform. |
+| Open Telemetry Collector | Replace | Vector implements the OT protocol with better performance, reliability, and data quality. |
+| Splunk UF | Replace | Vector integrates deeply with Splunk with better performance, reliability, and data quality. |
+| Syslog | Replace | Vector implements the Syslog protocol with better performance, reliability, and data quality. |
+| Telegraf | Integrate | Telegraf integrates with more systems and produces better metrics data. |
+
+#### **Integrating with Agents**
```

Comment:
It seems weird to have this emphasis for headings (here and elsewhere). Did you mean to include it? I think we should just use the normal heading style.